internal/syncer/connector/slack.go

```
@@ -0,0 +1,974 @@
+	"channel_posting_permissions": {},
+	"group_join":                  {},
+	"group_leave":                 {},
+	"group_archive":               {},
+	"group_unarchive":             {},
+	"channel_name":                {},
+}
+
+// acceptSlackMessage reports whether a message should be indexed.
+func acceptSlackMessage(message slackMessage) bool {
+	if message.BotID != "" || message.AppID != "" {
+		if message.BotProfile.Name == "DanswerBot Testing" {
```

golangci-lint (2.12.2)

[error] S1008: should use 'return message.BotProfile.Name == "DanswerBot Testing"' instead of 'if message.BotProfile.Name == "DanswerBot Testing" { return true }; return false'

(staticcheck)